core/domain/suggestion_services_test.py

```diff
@@ -7530,33 +7530,29 @@ def test_create_question_contributor_certificate_with_image_content(
     def test_create_contributor_certificate_raises_exception_for_no_suggestions(
         self
     ) -> None:
-        with self.assertRaisesRegex(
-            Exception,
-            'There are no contributions for the given time range.'
-        ):
-            suggestion_services.generate_contributor_certificate_data(
+        data = suggestion_services.generate_contributor_certificate_data(
                 self.username,
```

Comment:
Indent this and the following by 4 rather than 8 from the previous line; ditto below.

Reply:
Done!